docs/service-mesh/03-01-istio-setup-in-kyma.md

```diff
@@ -33,7 +33,7 @@ These configuration changes are applied to customize Istio for use with Kyma:
 - [Mutual TLS (mTLS)](https://istio.io/docs/concepts/security/#mutual-tls-authentication) is enabled cluster-wide in a STRICT mode.
 - Global tracing is set to use the Zipkin installation provided by Kyma.
 - Ingress Gateway is expanded to handle ports `80` and `443` for local Kyma deployments.
-- DestinationRules are created by default, which disables mTLS for the `kubernetes.default.svc.cluster.local` service. In local (Minikube) installation mTLS is also disabled for
+- In local (Minikube) installation there is a DestinationRule that disables mTLS for
 `istio-ingressgateway.istio-system.svc.cluster.local` service.
```

Comment:
We can move that piece to the line above as it's one sentence. See the above comment.